src/sqs/push.ts

```diff
@@ -39,3 +41,60 @@ export const pushQueueMessageHandler : MessageHandler<PushQueueMessagePayload> =
 	await processPush(github, payload, wrapLogger(context.log));
 }
 
+
+const NON_ERRONEOUS_STATUSES = [401, 404, 403];
+const RATE_LIMITING_DELAY_BUFFER = 10;
+const EXPONENTIAL_BACKOFF_BASE = 60;
+export const pushQueueErrorHandler : ErrorHandler<PushQueueMessagePayload> = async (error: JiraClientError | Octokit.HookError | RateLimitingError | Error,
+	context: Context<PushQueueMessagePayload>) : Promise<ErrorHandlingResult> => {
+
+	//Handle non-failure cases
+	const maybeResult = maybeHandleNonFailureCase(error, context);
+	if(maybeResult) {
```

Comment:
supernit: here and in other cases: there should either be a space between `if` and `(` everywhere or nowhere. I see both in different parts of your PR

Reply:
Looks like we need to add some liner rules so it will be auto-formatted